src/js/videojs-dash.js

```diff
@@ -73,11 +73,18 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
-    // Apply any options that are set
-    if (options.dash && options.dash.limitBitrateByPortal) {
```

Comment:
@forbesjo Can you please clarify what you mean? If the user uses `limitBitrateByPortal`, it will still work the same as it did before if this update gets merged.